kitsune/users/static/users/js/constants/UserAuthConstants.es6.js

```diff
@@ -1,27 +0,0 @@
-import constantMap from '../../../sumo/js/utils/constantMap.es6.js';
```

Comment:
Referenced in kitsune/questions/static/questions/js/components/UserAuth.jsx

and kitsune/questions/static/questions/js/components/SubmitQuestion.jsx

and kitsune/questions/static/questions/js/components/AttachmentField.jsx